src/migrations/2017-08-10-organization_uuid.js

```diff
@@ -0,0 +1,10 @@
+import { r } from '../server/models'
+
+async function migrate() {
```

Comment:
ok so the idea is to have a uuid and an id, but just not expose the id in the url?